guillotina/_settings.py

```diff
@@ -91,5 +91,7 @@
     "search_parser": "default",
     "object_reader": "guillotina.db.reader.reader",
     "thread_pool_workers": 32,
+    "keep_alive": 5,
+    "http_protocol": "h11"
```

Comment:
it's a shame we end up with h11 here

Reply:
Its temporal until encode/uvicorn#344 is fixed